src/Processor/ValidateUserInput.cs

```diff
@@ -166,6 +167,18 @@ private static bool IsWorkflowValid(UserInput userInputObj)
             // Validate Assessment settings for Assessment module;
             return ValidateAssessmentSettings(userInputObj);
         }
+        
+        private static bool IsBusinessProposalValid(UserInput userInputObj)
+        {
+            string businessProposal = "";
+            if (userInputObj.IsAvsProposal)
+                businessProposal = "Quick AVS Proposal";
+            else
+                businessProposal = "Comprehensive Proposal for IaaS, PaaS and AVS targets​";

```

Comment:
Enum?

Reply:
used the same format as isExpressWorkflow